### ISSUE TITLE
Testcase in SparkBigQueryEndToEndReadITSuite fails

### ISSUE THREAD

**Issue by animeshkash-db** [GoogleCloudDataproc/spark-bigquery-connector#346]
Recently `default number of partitions. DataSource %s. Data Format %s` testcase of `SparkBigQueryEndToEndReadITSuite` has started to fail (https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/master/connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndReadITSuite.scala#L201).

Error:
```
[info] - default number of partitions. DataSource bigquery. Data Format avro *** FAILED ***
[info]   58 did not equal 35 (SparkBigQueryEndToEndReadITSuite.scala:207)
[info] - default number of partitions. DataSource com.google.cloud.spark.bigquery.v2.BigQueryDataSourceV2. Data Format avro *** FAILED ***
[info]   58 did not equal 35 (SparkBigQueryEndToEndReadITSuite.scala:207)
[info] - default number of partitions. DataSource bigquery. Data Format arrow *** FAILED ***
[info]   58 did not equal 35 (SparkBigQueryEndToEndReadITSuite.scala:207)
[info] - default number of partitions. DataSource com.google.cloud.spark.bigquery.v2.BigQueryDataSourceV2. Data Format arrow *** FAILED ***
[info]   58 did not equal 35 (SparkBigQueryEndToEndReadITSuite.scala:207)
```

It looks like the default number of streams for reading `bigquery-public-data.samples.natality` file as returned by the backend (https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/master/connector/src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala#L144) has changed from 35 to 58, and due to this the number of partitions also changes.

**Comment by davidrabinowitz** [comment]
Thanks, I'll check it out

**Comment by davidrabinowitz** [comment]
Fixed by PR #347
